tectonic/XeTeXFontInst.cpp

```diff
@@ -315,6 +315,9 @@ XeTeXFontInst::initialize(const char* pathname, int index, int &status)
     TT_OS2* os2Table;
     FT_Error error;
     hb_face_t *hbFace;
+    size_t sz;
+    ssize_t r;
+    FT_Byte *data;
```

Comment:
Ok. It is just an habit, in my case I prefer to have variables declared at the beginning of the current scope :) I will go back to the previous declaration.